libs/hpc-ui/src/lib/components/main-navigation.tsx

```diff
@@ -90,8 +95,7 @@ const Nav = styled.nav`
       position: relative;
       z-index: 1;
       display: flex;
-      align-items: center;
-      height: ${HEADER_HEIGHT_PX}px;
+      height: 36px;
```

Comment:
This is the only change you need. Amend the commit to only include this change. I was skeptical of using pixels as units, but they are used throughout the surrounding styles, so it's fine.